WidgetBus/Views/SelectStartNodeView/SelectStartNodeViewController.swift

```diff
@@ -0,0 +1,269 @@
+//
+//  SelectStartNodeViewController.swift
+//  WidgetBus
+//
+//  Created by 김민재 on 2022/11/15.
+//
+
+import UIKit
+
+class SelectStartNodeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
```suggestion
final class SelectStartNodeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

더이상 상속되지않는다는 표시가 필요해보입니다

Reply:
좋은 지적 감사합니다.